matscipy/pressurecoupling.py

```diff
@@ -396,19 +396,19 @@ class SlideLog(object):
 
     The data of the log files is found as attributes
     (numpy arrays with step as axis):
-    step -- step indices 0, 1, 2, ...
-    time -- simulation time in fs at step
-    T_thermostat -- instantaneous temperature in K from thermostatet region
+    step : step indices 0, 1, 2, ...
```

Comment:
Please update to numpydoc, see here: https://numpydoc.readthedocs.io/en/latest/format.html

```
step : int
    Step indices 0, 1, 2, ...
```

The data type follows the colon, not the description. The description is in the next line indented by four spaces. Please update.

Reply:
Thanks for the review, should be ok now.